plugins/bazaar/src/components/BazaarInfoCard/BazaarInfoCard.test.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { screen, cleanup } from '@testing-library/react';
+import { BazaarInfoCard } from './BazaarInfoCard';
+import { renderInTestApp } from '@backstage/test-utils';
+
+afterEach(() => {
+  cleanup();
+});
+
+it('should render empty ProjectPreview', async () => {
+  await renderInTestApp(<BazaarInfoCard />);
+  const cardElement = screen.getByTestId('empty-bazaar');
+  expect(cardElement).toBeInTheDocument();
+  expect(cardElement).toHaveTextContent('Please add projects to the Bazaar');
```

Comment:
```suggestion
  const rendered = await renderInTestApp(<BazaarInfoCard />);
  expect(rendered.getByText('Please add projects to the Bazaar')).toBeInTheDocument();
```

This test doesn't seem functional as-is, just wanted to show a little more idiomatic testing. 😅  `getByTestId` should be a last resort, testing via what the user sees instead is the philosophy behind `testing-library`. And one should use the return from `render` / `renderInTestApp` rather than `screen`, where possible.